lib/isexpired.c

```diff
@@ -22,6 +22,24 @@
 
 #ident "$Id$"
 
+/*
+ * addsl - adds three non-negative long values
+ *
+ *	the result is capped at LONG_MAX.
```

Comment:
I suggest using the word saturate instead of cap, since the function name uses an s.

Reply:
Adjusted